app/views/items/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% @items.each do |item| %>
+  <p>name: <%= item.name %></p>
+  <p>price: <%= item.price %></p>
+  <p>Supermarket: <%= item.supermarket.name %></p>
+  <p>Customers: <%= item.customers.count %></p>
```

Comment:
For things like line 4 and 5, we'll want to create model methods to house this logic so from the view, we're only calling something like `item.supermarket_name` and `item.customer_count`. I know it's a small tweak in this case, but as our queries get more complex, we want all that logic to live in the models.